src/components/ui/carousel.tsx

```
@@ -0,0 +1,262 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)